<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CViewQuit</class>
 <widget class="QWidget" name="CViewQuit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CViewQuit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>界面程序</string>
     </property>
     <widget class="QPushButton" name="pushButtonResetHmi">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>22</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>重启(&amp;R)</string>
      </property>
      <property name="icon">
       <iconset resource="ssp_gui.qrc">
        <normaloff>:/ssp_gui/Resources/hujiaozhuanyi.gif</normaloff>:/ssp_gui/Resources/hujiaozhuanyi.gif</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>72</y>
        <width>551</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonQuitHmi">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退出(&amp;Q)</string>
      </property>
      <property name="icon">
       <iconset resource="ssp_gui.qrc">
        <normaloff>:/ssp_gui/Resources/W95MBX01.ICO</normaloff>:/ssp_gui/Resources/W95MBX01.ICO</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>34</y>
        <width>551</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>注：重启命令将使界面程序退出，并立即再次启动</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>操作系统</string>
     </property>
     <widget class="QPushButton" name="pushButtonResetSys">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>22</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>重启(&amp;B)</string>
      </property>
      <property name="icon">
       <iconset resource="ssp_gui.qrc">
        <normaloff>:/ssp_gui/Resources/KEY04.ICO</normaloff>:/ssp_gui/Resources/KEY04.ICO</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>72</y>
        <width>551</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>注：关机操作将中止所有程序并关闭操作系统，关机后需要手动关闭装置电源</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonHaltSys">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>关机(&amp;H)</string>
      </property>
      <property name="icon">
       <iconset resource="ssp_gui.qrc">
        <normaloff>:/ssp_gui/Resources/MSGBOX01.ICO</normaloff>:/ssp_gui/Resources/MSGBOX01.ICO</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>34</y>
        <width>551</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>注：操作系统重启命令将中止所有程序退出操作系统，并立即再次启动操作系统</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>176</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ssp_gui.qrc"/>
 </resources>
 <connections/>
</ui>
